handlers/delete_lease.go

```diff
@@ -48,17 +46,17 @@ func DeleteLease(services k8s.ServiceGetterUpdater, k8sServiceName string, names
 		}
 
 		// clear all namespaces
-		namespacesList, err := namespaces.List(k8sapi.ListOptions{LabelSelector: labels.Everything()})
-		if err != nil {
-			htp.Error(w, http.StatusInternalServerError, "cannot get namespaces (%s)", err)
-			return
-		}
-		// TODO: delete concurrently https://github.com/deis/k8s-claimer/issues/49
-		for _, namespace := range namespacesList.Items {
-			if namespace.Name != "kube-system" && namespace.Name != "default" {
-				namespaces.Delete(namespace.Name)
-			}
-		}
+		// namespacesList, err := namespaces.List(k8sapi.ListOptions{LabelSelector: labels.Everything()})
+		// if err != nil {
+		// 	htp.Error(w, http.StatusInternalServerError, "cannot get namespaces (%s)", err)
+		// 	return
+		// }
+		// // TODO: delete concurrently https://github.com/deis/k8s-claimer/issues/49
+		// for _, namespace := range namespacesList.Items {
+		// 	if namespace.Name != "kube-system" && namespace.Name != "default" {
+		// 		namespaces.Delete(namespace.Name)
+		// 	}
+		// }
```

Comment:
@jchauncey if you want, you can just delete this code. There's a fix in progress in #68, and the code will change significantly anyway